src/mongodb/config/language/or.ts

```
@@ -0,0 +1,6 @@
+var or_config = {
+    vowel: ["ା", "ି", "ୀ", "ୁ", "ୂ", "ୃ", "ୄ", "େ", "ୈ", "ୋ", "ୌ", "ઃ"],
+    language_code: "or"
+};
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)